src/test/java/info/novatec/camunda/migrator/ProcessInstanceMigratorTest.java

```diff
@@ -310,6 +309,75 @@ public void processInstanceMigrator_should_not_migrate_to_higher_major_version()
             .allTasksHaveDefinitionId(newestProcessDefinition.getId())
             .allTasksHaveFormkey(null);
     }
+    
+    @Test
+    public void processInstanceMigrator_should_not_migrate_process_instances_to_models_without_version_tag() {
+    	deployBPMNFromClasspathResource(NON_MIGRATEABLE_PROCESS_MODEL_WITHOUT_VERSION);
+        ProcessDefinition newestProcessDefinitionAfterRedeployment = getNewestDeployedProcessDefinitionId(PROCESS_DEFINITION_KEY);
+        assertThat(newestProcessDefinitionAfterRedeployment.getVersionTag()).isEqualTo(null);
+
+        assertThat(getRunningProcessInstances(PROCESS_DEFINITION_KEY))
+            .numberOfProcessInstancesIs(2)
+            .allProcessInstancesHaveDefinitionId(newestProcessDefinition.getId());
+
+        assertThat(getCurrentTasks(PROCESS_DEFINITION_KEY))
+            .numberOfTasksIs(2)
+            .allTasksHaveDefinitionId(newestProcessDefinition.getId())
+            .allTasksHaveFormkey(null);
+
+        processInstanceMigrator.migrateProcessInstances(PROCESS_DEFINITION_KEY);
+
+        assertThat(getRunningProcessInstances(PROCESS_DEFINITION_KEY))
+            .numberOfProcessInstancesIs(2)
+            .allProcessInstancesHaveDefinitionId(newestProcessDefinition.getId());
+
+        assertThat(getCurrentTasks(PROCESS_DEFINITION_KEY))
+            .numberOfTasksIs(2)
+            .allTasksHaveDefinitionId(newestProcessDefinition.getId())
+            .allTasksHaveFormkey(null);
+    }
+    
+    @Test
+    public void processInstanceMigrator_should_not_fail_if_only_process_models_without_version_tag_exist() {
+    	cleanUp();
```

Comment:
wouldn't it be nicer to generally have the "cleanup" called via `@Before`?

Reply:
The whole construct was a little dirty. I refactored the tests somewhat to make the setup more consistent. Cleanup is and will be called via @After.